extensions/wikia/AdEngine/js/template/porvata.js

```diff
@@ -17,7 +28,25 @@ define('ext.wikia.adEngine.template.porvata', [
 	 * @param {string} [params.vastUrl] - Vast URL (DFP URL with page level targeting will be used if not passed)
 	 */
 	function show(params) {
+		if (params.vpaidMode === 2) {
```

Comment:
can we create a const ie INSECURE_MODE = 2; and then use it across the file instead of comparing to 2?